script/download-grammars

```diff
@@ -139,7 +139,7 @@ def load_grammar(path)
   case File.extname(path.downcase)
   when '.plist', '.tmlanguage'
     Plist::parse_xml(path)
-  when '.cson'
+  when '.cson', '.json'
     cson = `"#{CSONC}" "#{path}"`
```

Comment:
There's no need to pass `.json` files through `csonc`. It would be slightly nicer to just go directly to `JSON.parse` in this case.